<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quit_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minimized_Btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="big_icon">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>215</y>
      <width>220</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="background_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>801</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="background_right">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>971</width>
      <height>801</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>120</y>
      <width>251</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="typeList_layout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="Type_Line" name="delta_uniform" native="true"/>
     </item>
     <item>
      <widget class="Type_Line" name="robustness" native="true"/>
     </item>
     <item>
      <widget class="Type_Line" name="fixed_ans_opssite" native="true"/>
     </item>
     <item>
      <widget class="Type_Line" name="branch_number" native="true"/>
     </item>
     <item>
      <widget class="Type_Line" name="transparence_order" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="InfoView" name="info_view" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>420</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="AnimatedLabel" name="back_Btn">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>520</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exe_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>test.exe</string>
    </property>
   </widget>
   <widget class="QLabel" name="process_view">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>110</y>
      <width>241</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="typeList_layout_2">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="Type_Line" name="algebraic_immunity" native="true"/>
     </item>
     <item>
      <widget class="Type_Line" name="get_snr_dpa" native="true"/>
     </item>
     <item>
      <widget class="Type_Line" name="balance" native="true"/>
     </item>
     <item>
      <widget class="Type_Line" name="nonlinearity" native="true"/>
     </item>
     <item>
      <widget class="Type_Line" name="correlation_immunity" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="Type_Line" name="algebraic_degree" native="true">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>239</width>
      <height>92</height>
     </rect>
    </property>
   </widget>
   <widget class="Type_Line" name="GAC" native="true">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>600</y>
      <width>239</width>
      <height>92</height>
     </rect>
    </property>
   </widget>
   <widget class="Type_Line" name="propagation_characteristics" native="true">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>690</y>
      <width>239</width>
      <height>92</height>
     </rect>
    </property>
   </widget>
   <widget class="Type_Line" name="computeKappaCPA" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>251</width>
      <height>92</height>
     </rect>
    </property>
   </widget>
   <zorder>background_left</zorder>
   <zorder>background_right</zorder>
   <zorder>quit_Btn</zorder>
   <zorder>minimized_Btn</zorder>
   <zorder>big_icon</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>info_view</zorder>
   <zorder>back_Btn</zorder>
   <zorder>exe_name</zorder>
   <zorder>process_view</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>algebraic_degree</zorder>
   <zorder>GAC</zorder>
   <zorder>propagation_characteristics</zorder>
   <zorder>computeKappaCPA</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedLabel</class>
   <extends>QLabel</extends>
   <header>animatedlabel.h</header>
  </customwidget>
  <customwidget>
   <class>Type_Line</class>
   <extends>QWidget</extends>
   <header>type_line.h</header>
   <container>1</container>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>InfoView</class>
   <extends>QWidget</extends>
   <header>infoview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
